Gopkg.toml

```diff
@@ -34,15 +34,39 @@
 
 [[override]]
   name = "k8s.io/kubernetes"
-  version = "=v1.8.0"
+  branch = "release-1.10"
+
+[[override]]
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.10.0"
+
+[[override]]
+  name = "k8s.io/api"
+  branch = "release-1.10"
+
+[[override]]
+  name = "k8s.io/apimachinery"
+  branch = "release-1.10"
+
+[[override]]
+  name = "k8s.io/apiserver"
+  branch = "release-1.10"
+
+[[override]]
+  name = "k8s.io/apiextensions-apiserver"
+  branch = "release-1.10"
 
 [[override]]
   name = "github.com/imdario/mergo"
   revision = "6633656539c1639d9d78127b7d47c622b5d7b6dc"
 
+[[override]]
+  name = "github.com/russross/blackfriday"
+  version = "v1.4-2-g300106c"
+
 [[override]]
   name = "google.golang.org/grpc"
-  version = "=v1.2.1"
+  version = "1.7.2"
```

Comment:
do we need to override the grpc version or even have this anymore since we don't have a server side component?

Reply:
Good point. I had to add this to fix up some version conflicts but I'll see if I can just remove it

Reply:
Actually it looks like google.golang.org/grpc is still needed in Draft for `draft delete`: https://github.com/Azure/draft/blob/974526fbf0e7d7ece543519effe172d078c64d11/cmd/draft/delete.go#L100

Helm bumped up from GRPC 1.2.1 to 1.7.2 in 2.8 or 2.9 that had some compatibility breakers, and because kubernetes or Helm don't use dep we still need to define what version to use or else 1.2.1 will be satisfied since that will satisfy dep's SAT when analyzing Draft's usage of GRPC in the codebase.

I did a quick look through the rest of Gopkg.toml and the rest all look like packages we still need/require for Draft to work. Thanks for checking this though! :)